harness/determined/common/experimental/trial.py

```diff
@@ -18,6 +18,37 @@ def _to_bindings(self) -> bindings.v1LogLevel:
         return bindings.v1LogLevel(self.value)
 
 
+_csb = bindings.v1GetTrialCheckpointsRequestSortBy
+
+
+class CheckpointSortBy(enum.Enum):
```

Comment:
Just for my own benefit: what's the idea behind creating new enums like `CheckpointSortBy` which seem to copy over information already held elsewhere (`bindings.v1GetTrialCheckpointsRequestSortBy`)? Not sure what is going on with these.

Reply:
we don't want to expose bindings.* to the user. imagine, you have to import `bindings.v1GetTrialCheckpointsRequestSortBy` to specify sort by? we create new enums to handle internal nasties like long names, fields in the db we don't want to expose, etc.